detector/weakcredentials/filebrowser/filebrowser.go

```diff
@@ -166,6 +166,7 @@ func checkLogin(ctx context.Context, fileBrowserIP string, fileBrowserPort int)
 	client := &http.Client{Timeout: requestTimeout}
 	targetURL := fmt.Sprintf("http://%s:%d/api/login", fileBrowserIP, fileBrowserPort)
 
+	//nolint:errchkjson
```

Comment:
I'd say we should check for the error here too

Reply:
this is marshalling a static struct, so the only way it could actually error would be if someone changed the struct to include an external value.

`errchkjson` is designed to account for that where possible, so it'll tell you when you should start error checking but it expects you to pass a struct rather than a `map` - I might actually open an issue with them as I think it should be safe to also ignore `map[string]string` with static values

Reply:
Ah, fair, didn't notice it was a static struct. Let's add a short comment about that after the nolint directive in that case.